src/__experimental__/components/checkbox/checkbox.style.js

```diff
@@ -0,0 +1,210 @@
+import styled, { css } from 'styled-components';
+import PropTypes from 'prop-types';
+import baseTheme from '../../../style/themes/base';
+import FieldHelpStyle from '../field-help/field-help.style';
+import LabelStyle from '../label/label.style';
+import StyledHelp from '../help/help.style';
+import { THEMES } from '../../../style/themes';
+
+const StyledCheckbox = styled.div`
+  padding-top: 8px;
+
+  input[type=checkbox] {
+    cursor: pointer;
+    opacity: 0;
+    margin: 0;
+    z-index: 999;
+  }
+
+  svg {
+    background-color: ${baseTheme.colors.white};
+    border: solid 1px ${baseTheme.colors.border};
```

Comment:
you should use `theme.colors.white`, as you have already set `baseTheme` as a default theme

Reply:
Done 👍